qa/tasks/rook.py

```diff
@@ -646,4 +646,28 @@ def task(ctx, config):
             yield
 
         finally:
+            to_remove = []
+            ret = _shell(ctx, config, ['ceph', 'orch', 'ls', '-f', 'json'], stdout=BytesIO())
+            if ret.exitstatus == 0:
+                r = json.loads(ret.stdout.getvalue().decode('utf-8'))
+                for service in r:
+                    removal_name = None
+                    if service['service_type'] == 'rgw':
+                        removal_name = 'rgw.' + service['spec']['rgw_realm']
```

Comment:
is this really needed? it seems like we should be able to just do ``if service['service_type'] in ['rgw', 'mds', 'nfs']:`` and remove ``service['service_name']``